metricbeat/module/prometheus/query/_meta/testdata/docs.json-expected.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
All files under `tesdata` directory are used for testing. In this a `docs.plain` file is needed which then will be used as a testing input. See for example https://github.com/elastic/beats/blob/a174425cb45c406a748b290dd2ef892c731cdac7/metricbeat/module/prometheus/collector/_meta/testdata/docs.plain.